lib/web_console/railtie.rb

```diff
@@ -37,6 +37,9 @@ class Railtie < ::Rails::Railtie
     end
 
     initializer 'web_console.mount_point' do
+      if root = Rails.application.config.relative_url_root
```

Comment:
`root` should be a `Pathname`, right? Can we call `root.join(Middleware.mount_point)` below? Also, can we test what happens if both `config.relative_url_root` and `config.web_console.mount_point` are set?